src/Common/Mailer.php

```diff
@@ -360,4 +388,4 @@ public function send($id)
             Model::triggerEvent('Core', 'after_email_sent', array('id' => $id));
         }
     }
-}
```

Comment:
No need to remove the break on the end of the file. All files in Fork end with an empty line. (You can probably set this as a setting in your editor, in Sublime Text, it's "ensure_newline_at_eof_on_save": true)